packages/dashboard/src/components/Wizard/Wizard.Base.tsx

```diff
@@ -19,33 +20,41 @@ export class WizardBase extends React.Component<IWizardProps, {}> {
       throw new Error('Wizard must have atleast one step.');
     }
 
-    const classNames = getClassNames(this.props.styles!, { theme: this.props.theme! });
-
     // if the step to render is already passed in, use that
     const wizardStepProps = this.props.stepToShow ? this.props.stepToShow : getStepToShow(this.props);
 
+    const classNames = getClassNames(this.props.styles!, {
+      theme: this.props.theme!,
+      isSubStep: wizardStepProps.isSubStep!,
+      isFirstSubStep: wizardStepProps.isFirstSubStep!
```

Comment:
Are isFirstSubStep and isSubStep actually optional? Should we provide default values instead and simplify the typings?

Reply:
These props are not directly on top of IWizardProps, but in IWizardStepProps, which is array prop inside.